### PR TITLE
Add examples for font-variant and font-variant-caps CSS property

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds an example for [`font-variant`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant) and [`font-variant-caps`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-caps). Let me know if you want to see any changes. Thanks!

### CODE REVIEW

File: live-examples/css-examples/fonts/font-variant.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@font-face{
```
Comment (wbamberg) [inline live-examples/css-examples/fonts/font-variant.css]: Super-nit: space before `{`

Comment (ddbeck) [reply live-examples/css-examples/fonts/font-variant.css]: Fixed, thank you!

File: live-examples/css-examples/fonts/font-variant-caps.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@font-face{
```
Comment (schalkneethling) [inline live-examples/css-examples/fonts/font-variant-caps.css]: Nit, code style: I reckon you just overlooked this one 👁

Comment (ddbeck) [reply live-examples/css-examples/fonts/font-variant-caps.css]: Yeah I did. I just pushed a change to find-and-replace the problem everywhere. :-)

**Pull request merged**